### ISSUE TITLE
typo in french [revenue -> revenu]

### ISSUE THREAD

**Issue by romjerome** [Diacamma2/syndic#2]
"Revenue" sounds English.

It seems that the french translation should use:

```diff
#: migrations/0001_initial.py:46
msgid "revenue"
-msgstr "revenue"
+msgstr "revenu"
```

../diacamma/syndic/locale/fr/LC_MESSAGES/django.po

Need to also fix "financial" module.

**Comment by GayLaurent** [comment]
Thanks ;)

**Comment by romjerome** [comment]
> ../diacamma/syndic/locale/fr/LC_MESSAGES/django.po
> 
> Need to also fix "financial" module.

This seems to be fixed with recent pull requests (#4 #5)
